codecov.yml

```diff
@@ -28,17 +27,9 @@ flag_management:
         target: 80%
   individual_flags:
     - name: unit-tests
-    - name: e2e-tests
-      # Wait until all machines have reported coverage - e2e tests run across 4 machines.
-      after_n_builds: 4
-      statuses:
-        - type: patch
-          target: 0%
 
 comment:
   layout: flags
-  # Wait until all machines have reported coverage - e2e tests run across 4 machines + unit tests across 1.
```

Comment:
just curious are we not waiting anymore or is this just not necessary because we wait somewhere else

Reply:
This PR removes E2E codecov, so the awaited tests no longer run.